cmd/package.go

```diff
@@ -0,0 +1,340 @@
+	snapcraftFile.WriteString("  app:\n")
+	snapcraftFile.WriteString("    plugin: dump\n")
+	snapcraftFile.WriteString("    source: build/outputs/linux\n")
+	snapcraftFile.WriteString("    stage-packages:\n")
+	for _, dependency := range linuxPackageDependencies {
+		snapcraftFile.WriteString("      - " + dependency + "\n")
+	}
+	snapcraftFile.WriteString("  desktop:\n")
+	snapcraftFile.WriteString("    plugin: dump\n")
+	snapcraftFile.WriteString("    source: snap\n")
+	snapcraftFile.WriteString("  assets:\n")
+	snapcraftFile.WriteString("    plugin: dump\n")
+	snapcraftFile.WriteString("    source: assets\n")
+	snapcraftFile.WriteString("apps:\n")
+	snapcraftFile.WriteString("  " + projectName + ":\n")
+	snapcraftFile.WriteString("    command: " + projectName + "\n")
+	snapcraftFile.WriteString("    desktop: local/" + projectName + ".desktop\n")
+	snapcraftFile.Close()
+
+	splitProjectName := strings.Split(projectName, "")
+	splitProjectName[0] = strings.ToUpper(splitProjectName[0])
+	titledProjectName := strings.Join(splitProjectName, "")
+	desktopFilePath, err := filepath.Abs(filepath.Join(snapLocalDirectoryPath, projectName+".desktop"))
+	if err != nil {
+		fmt.Printf("hover: Failed to resolve absolute path for desktop file %s: %v\n", desktopFilePath, err)
+		os.Exit(1)
+	}
+	desktopFile, err := os.Create(desktopFilePath)
+	if err != nil {
+		fmt.Printf("hover: Failed to create desktop file %s: %v\n", desktopFilePath, err)
+		os.Exit(1)
+	}
+	desktopFile.WriteString("[Desktop Entry]\n")
+	desktopFile.WriteString("Encoding=UTF-8\n")
+	desktopFile.WriteString("Version=" + assertInFlutterProject().Version + "\n")
+	desktopFile.WriteString("Type=Application\n")
+	desktopFile.WriteString("Terminal=false\n")
+	desktopFile.WriteString("Exec=/" + projectName + "\n")
+	desktopFile.WriteString("Name=" + titledProjectName + "\n")
+	desktopFile.WriteString("Icon=/icon.png\n")
+	desktopFile.Close()
+
+	fmt.Println("hover: Packaging snap...")
+	cmdBuildSnap := exec.Command(snapcraftBin)
+	cmdBuildSnap.Dir = filepath.Join(snapDirectoryPath, "..")
+	cmdBuildSnap.Stdout = os.Stdout
+	cmdBuildSnap.Stderr = os.Stderr
+	cmdBuildSnap.Stdin = os.Stdin
+	cmdBuildSnap.Start()
+	cmdBuildSnap.Wait()
+
+	outputFilePath := filepath.Join(outputDirectoryPath(targetOS), projectName+"_"+runtime.GOARCH+".snap")
+	os.Rename(filepath.Join(snapDirectoryPath, "..", projectName+"_"+assertInFlutterProject().Version+"_"+runtime.GOARCH+".snap"), outputFilePath)
+	if install {
+		fmt.Println("hover: Installing snap...")
+		var snapBin, err = exec.LookPath("snap")
+		if err != nil {
+			fmt.Println("hover: Failed to lookup `snap` executable. Please install snap.")
+			os.Exit(1)
+		}
+		cmdInstallSnap := exec.Command("/bin/sh", "-c", "sudo "+snapBin+" install "+outputFilePath+" --devmode")
+		cmdInstallSnap.Stdout = os.Stdout
+		cmdInstallSnap.Stderr = os.Stderr
+		cmdInstallSnap.Stdin = os.Stdin
+		cmdInstallSnap.Start()
+		cmdInstallSnap.Wait()
```

Comment:
Use `cmdInstallSnap.Run()` (it combines Start and Wait). Also: check for errors.